### ISSUE TITLE
Website is confusing, with lots of "Noddity" blog posts.

### ISSUE THREAD

**Issue by Jasha10** [plotters-rs/plotters-rs.github.io#4]
The plotters git repo links to the plotters website https://plotters-rs.github.io/home/#!/
The website has lots of links to blog posts about Noddity, which seems unrelated to the plotters software.
 
![plotters-rs github io_home_](https://github.com/plotters-rs/plotters-rs.github.io/assets/8935917/0669511d-8732-4a7c-8ada-d8ef85ca1d23)

Besides cleaning up the Noddity links, another potential improvement would be to link to the plotters developer guide, https://plotters-rs.github.io/book/.

**Comment by jacobsvante** [comment]
Hacked?

**Comment by Jasha10** [comment]
> Hacked?

I don't think so. The noddity blog posts are actually hosted here in this repo, e.g.:
https://github.com/plotters-rs/plotters-rs.github.io/blob/22f1c9ae2d574d3a67bf1a070506967e475fa870/home/content/contribute.md#L6

My hypothesis is that this noddity material was copy-pasted into this repo by accident.
